samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -247,7 +247,7 @@ After programming the sample to your development kit, complete the following ste
       #. Reset the kit.
       #. Observe that **LED 0** is blinking and the device is advertising under the default name **Nordic_UART_Service**.
          You can configure this name using the :kconfig:option:`CONFIG_BT_DEVICE_NAME` Kconfig option.
-      #. Observe that the text "Starting Nordic UART service example" is printed on the COM listener running on the computer.
+      #. Observe that the text "Starting Nordic UART service sample" is printed on the COM listener running on the computer.
```

Comment:
Here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/rpc_host/README.html#testing, 
 Shouldn't the Testing section of the rpc_host sample also updated with "sample" instead of example.

_After programming the example build to your ........_

Reply:
Good catch! I missed that even though I run a search to Bluetooth samples.

Reply:
Yeah, this also needs update!

Reply:
Fixed